test/Escrow.js

```diff
@@ -10,59 +11,46 @@ const hash = 'fakehash';
 const recOIpns = 'https://recO.io';
 const repOIpns = 'https://repO.com';
 
-contract('Escrow', (accounts) => {
+contract('Escrow', accounts => {
   beforeEach(async () => {
-    HMT = await HMTokenAbstraction.new('100', 'Human Token', 4, 'HMT', { from: accounts[0] });
+    HMT = await HMTokenAbstraction.new('100', 'Human Token', 4, 'HMT', {
+      from: accounts[0]
+    });
     reputationOracle = accounts[1];
     recordingOracle = accounts[2];
 
-    Escrow = await EscrowAbstraction.new(HMT.address, accounts[0], 5, { from: accounts[0] });
+    Escrow = await EscrowAbstraction.new(HMT.address, accounts[0], 5, {
+      from: accounts[0]
+    });
   });
 
   describe('calling getTokenAddress', () => {
     it('returns correct token address', async () => {
-      try {
-        const address = await Escrow.getTokenAddress.call();
-        assert.equal(address, HMT.address);
-      } catch (ex) {
-        assert(false);
-      }
+      const address = await Escrow.getTokenAddress();
+      assert.equal(address, HMT.address);
     });
   });
 
-  describe('initial', () => {
+  describe('initialization of Escrow contract works correctly', () => {
     it('status is launched', async () => {
-      try {
-        const initialStatus = await Escrow.getStatus.call();
-        assert.equal(0, initialStatus);
-      } catch (ex) {
-        assert(false);
-      }
+      const initialStatus = await Escrow.getStatus();
+      assert.equal(0, initialStatus);
```

Comment:
Switch over to `actual`, `expected`.

`assert.equal(initialStatus, 0)`

Reply:
Updated in [latest commit](https://github.com/hCaptcha/hmt-escrow/pull/130/commits/a276f2f8624e8aeb7be4266173fa33bb70c910dd#diff-6639d32f4f91562f4eec97bd924b494eR37) :handshake: